src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpoint.java

```diff
@@ -295,6 +295,28 @@ public CaseEndpoint(final CaseService caseService) {
     return ResponseEntity.ok(result);
   }
 
+  /**
+   * the GET end point to request a UAC from AD for a given caseid
+   *
+   * @param caseId the id of the case
+   * @param requestBodyDTO the request body
```

Comment:
can't find requestBodyDTO - should it be requestParamsDTO

Reply:
Yep, changed.